build_resolvers/lib/src/analysis_driver.dart

```diff
@@ -67,8 +67,8 @@ Packages _buildAnalyzerPackages(
       for (var package in packageConfig.packages)
         package.name: Package(
           name: package.name,
-          languageVersion: Version(
-              package.languageVersion.major, package.languageVersion.minor, 0),
+          languageVersion: Version(package.languageVersion?.major ?? 0,
+              package.languageVersion?.minor ?? 0, 0),
```

Comment:
It looks like analyzer allows a null version, which should handle the defaulting correctly, so this is I think the more semantically correct solution: 

```suggestion
          languageVersion: package.languageVersion == null
              ? null
              : Version(package.languageVersion.major,
                  package.languageVersion.minor, 0),
```